internal/acctest/authenticationpoliciessettings/authentication_policies_settings_test.go

```diff
@@ -0,0 +1,100 @@
+package acctest_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/acctest"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/provider"
+)
+
+// Attributes to test with. Add optional properties to test here if desired.
+type authenticationPoliciesSettingsResourceModel struct {
+	enableIdpAuthnSelection bool
+	enableSpAuthnSelection  bool
+}
+
+func TestAccAuthenticationPoliciesSettings(t *testing.T) {
+	resourceName := "myAuthenticationPoliciesSettings"
+	initialResourceModel := authenticationPoliciesSettingsResourceModel{
+		enableIdpAuthnSelection: false,
+		enableSpAuthnSelection:  true,
+	}
+	updatedResourceModel := authenticationPoliciesSettingsResourceModel{
+		enableIdpAuthnSelection: true,
+		enableSpAuthnSelection:  false,
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() { acctest.ConfigurationPreCheck(t) },
+		ProtoV6ProviderFactories: map[string]func() (tfprotov6.ProviderServer, error){
+			"pingfederate": providerserver.NewProtocol6WithError(provider.NewTestProvider()),
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAuthenticationPoliciesSettings(resourceName, initialResourceModel),
```

Comment:
To align with our other tests, the first step should set only the required properties of the resource. In this case there are no required properties, so the first step can just apply some HCL with no attributes in the resource itself. 
There should also be a 4th test step that returns to the initial empty model.

Reply:
There should be four steps. Minimal -> Maximal -> Import -> back to Minimal.